src/form/birth/index.ts

```diff
@@ -254,11 +255,16 @@ export const birthForm: ISerializedForm = {
               certificateHandlebars.informantNationality,
               hideIfInformantMotherOrFather
             ), // Required field.
-            getNationalID(
-              'informantID',
-              hideIfNidIntegrationEnabled.concat(hideIfInformantMotherOrFather),
-              getNationalIDValidators('informant'),
-              certificateHandlebars.informantNID
+            getIDType(
+              'birth',
+              'informant',
+              hideIfNidIntegrationEnabled.concat(detailsExist),
```

Comment:
`detailsExist` doesn't apply to informant so we need `hideIfInformantMotherOrFather` instead

Reply:
Good catch @Zangetsu101 I am going to correct that